app/views/volunteers/_form.html.erb

```diff
@@ -15,6 +15,5 @@
 
   <div class="actions">
     <%= form.submit "Create User", class: "btn btn-primary" %>
-    <%= return_to_dashboard_button %>
```

Comment:
Please also remove the implementation of the button if possible, or open a github "chore" issue to clean up after this PR :)

Reply:
Combined #2504 #2577 in this PR and removed the implementation as well